benefits/enrollment/views.py

```diff
@@ -177,38 +177,12 @@ def success(request):
 
     verifier = session.verifier(request)
 
-    page = viewmodels.Page(title=_("enrollment.pages.success.title"), headline=_("enrollment.pages.success.headline"))
+    page = viewmodels.Page()
```

Comment:
What happens if you don't create this `viewmodel.Page` and don't pass the `context_dict()` to the template?